packages/protocol/contracts/L1/tiers/TierProviderBase.sol

```diff
@@ -22,7 +22,7 @@ abstract contract TierProviderBase is ITierProvider {
                 contestBond: 500 ether, // TKO
                 cooldownWindow: 1440, //24 hours
                 provingWindow: 30, // 0.5 hours
-                maxBlocksToVerifyPerProof: 16
+                maxBlocksToVerifyPerProof: 0
```

Comment:
Proving a block would not trigger verification from this point on ?

Reply:
No, but this field is not used. We use TaikoL1.getConfig().maxBlocksToVerify instead regardless of tiers.